src/plots/cartesian/layout_attributes.js

```diff
@@ -721,6 +721,17 @@ module.exports = {
             'outside and vice versa.'
         ].join(' ')
     },
+    ticklabelindex: {
+        // In the future maybe add `arrayOk: true` for labeling several minor ticks, e.g. [-1, 0, 1]
```

Comment:
The PR looks very good.
Wondering by adding a loop if you could also introduce this feature as an `arrayOk: true` attribute?
Then the users could label e.g. Q2 and Q4 tick, etc.

Reply:
Yeah, it looks like it works. I added a test case.